scripts/build_base_image.sh

```diff
@@ -4,46 +4,32 @@ set -e
 function usage {
   echo "Usage: $0 [options]"
   echo "Options:"
-  echo "  -t, --tag <value>        Tag to add to image, repeatable"
+  echo "  -t, --tag <value>        Set the tag (default: latest)"
+  echo "  -v, --version <value>    Fondant version"
```

Comment:
What's the reason to split these? Looking at how they're used in the Github actions, I would expect it to work with a single value in all those cases?

Reply:
I've split it into two cause otherwise the order of the tags in the list is relevant. 
In the component build script the first tag is used to determine the version of fondant. 
E.g. defining `[...] -t GITHUB_HASH -t dev` would work, but not `[...] -t dev -t GITHUB_HASH`.